todo_app/view_model.py

```diff
@@ -9,7 +9,7 @@ def __init__(self, items):
     def todo_items(self):
         output = []
         for item in self._items:
-            if item.status == "ToDo":
+            if item.status == "To Do":
```

Comment:
Your update status button in index.html still sets status to ToDo

Reply:
Done